firefox-ios/firefox-ios-tests/Tests/XCUITests/PrivateBrowsingTest.swift

```diff
@@ -283,6 +283,17 @@ fileprivate extension BaseTestCase {
             "The number of tabs is not correct"
         )
     }
+
+    func enableClosePrivateBrowsingOptionWhenLeaving() {
```

Comment:
All tests' names should start with `test`. If this is a test, the name should be `testEnableClosePrivateBrowsingOptionWhenLeaving()`.

In addition, this test should be added to FullFunctionalTestPlan only. I don't see a change in the test plan file, so this test may be added to every test plan (not we wanted).

Reply:
Sorry. My bad. This is not a test.

Reply:
`enableClosePrivateBrowsingOptionWhenLeaving()` is not used anywhere at the moment. 😕

Reply:
If you find old tests that exercise this helper, please add the old tests back.